modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install

```diff
@@ -549,13 +549,14 @@ function social_group_flexible_group_update_11402(): string {
 function social_group_flexible_group_update_11701(array &$sandbox): string {
   if (!isset($sandbox['total'])) {
     $sandbox['total'] = \Drupal::entityQuery('group')
+      ->accessCheck(FALSE)
```

Comment:
Are we sure that the access check should be FALSE? Default is TRUE. Same for below

Reply:
We want to update all the groups and I do not want that any access check alter interfere with process. However, you you might have an example why we should consider to changing this to TRUE? :)

Reply:
Not really, just making sure we got the entire scenario covered.

Reply:
Not really, just making sure we got the entire scenario covered.